src/js/abstract/hocs/with-version.js

```diff
@@ -0,0 +1,22 @@
+/* globals VERSION */
+
+const withVersion = Base =>
+  /**
+   * Adds the current `version` from `package.json` to each component.
+   *
+   * @example
+   * const AXAButton = customElements.get('axa-button');
+   * console.log(AXAButton.version);
+   *
+   * // or by promise
+   * customElements.whenDefined('axa-button')
+   *   .then(() => {
+   *     const AXAButton = customElements.get('axa-button');
+   *     console.log(AXAButton.version);
+ *     });
+   */
+  class WithVersion extends Base {
+    static version = VERSION;
```

Comment:
Where does this `VERSION` come? I suppose the babel plugin read it out from the package json. But will this solve the requirement of "having more than one pattern lib with different version in that only one will be bundles"? https://github.com/axa-ch/midgard/issues/10

Reply:
from `package.json` at build time by https://www.npmjs.com/package/babel-plugin-version-transform

Reply:
And **NO** it will not solved bundling of different version.
This is about to ease debugging for us, that we can safely determine the version used of plib.